recipes-tag/11-id-chx-analysis/meta.yaml

```diff
@@ -7,10 +7,11 @@ package:
   version: {{ year }}{{ cycle }}.{{ version }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
     - chxtools
     - eiger-io
     - hdf5-lz4
+    - chxanalys
```

Comment:
@licode CHX needs these as well. Most are probably available from anaconda. Can you identify which (if any) need to be copied from conda-forge?